src/css/index.css

```diff
@@ -1,173 +1,745 @@
-  left: 241px;
   background: url(../images/header/play-bg.png) no-repeat center;
   opacity: 0.8;
+  display: flex;
+  justify-content: center;
+  align-items: center;
 }
 
 
 .play-btn {
-  position: absolute;
   width: 18px;
   height: 24px;
-  top: 24px;
-  left: 30px;
 }
+
+
+.header-down-arrow {
+  width: 16px;
+  height: 6.67px;
+  margin: auto;
+  margin-top: 52px;
+}
+
+
+/* ---------------------------------------------------------------------- *\
+                              #SECTION FEATURE
+\* ---------------------------------------------------------------------- */
+
+
+.features {
+  max-width: 1440px;
+  height: 768px;
+  background: var(--color-white);
+}
+
+
+.features-title {
+  text-align: center;
+  margin: 80px 526px;
+  color: var(--color-cstom);
+}
+
+
+.features-title-name {
+  margin: 0 100px;
+}
+
+
+.features-title-subtitle {
+  font-size: 18px;
+  margin-top: 32px;
+}
+
+
+.features-content {
+  margin: 32px 72px 59px 162px;
+  display: flex;
+}
+
+
+.features-content-left {
+  font-size: 16px;
+  line-height: 24px;
+}
+
+
+.responsive-layout, 
+.pixel-perfect, 
+.organized-layers {
+  display: grid;
+  grid-template-columns: 60px 262px;
+}
+
+
+.responsive-layout-content,
+.pixel-perfect-content,
+.organized-layers-content {
+  display: flex;
+  flex-direction: column;
+}
+
+
+.responsive-layout-content-title,
+.pixel-perfect-content-title,
+.organized-layers-content-title {
+  margin: 0 10px;
+}
+
+
+.responsive-layout-content-description,
+.pixel-perfect-content-description,
+.organized-layers-content-description {
+  margin: 10px 10px 30px 10px;
+}
+
+
+.features-content-right {
+  background: url(../images/features/mac.png);
+  margin-left: 100px;
+  width: 802px;
+  height: 464px;
+}
+
+
+/* ---------------------------------------------------------------------- *\
+                              #SECTION DOWNLOAD
+\* ---------------------------------------------------------------------- */
+
+
+.download {
+  max-width: 1440px;
+  height: 588px;
+  background: linear-gradient(0deg, var(--color-black-gray), var(--color-black-gray)), url(../images/download/background.png);
+  background-blend-mode: soft-light, normal;
+}
+
+
+.download-content {
+  text-align: center;
+  color: var(--color-white);
+  font-style: normal;
+  font-weight: normal;
+  padding-top: 163px;
+  padding-bottom: 42px;
+}
+
+
+.download-content-title {
+  margin-bottom: 20px;
+}
+
+
+.download-content-description {
+  width: 555px;
+  margin: 0 auto;
+}
+
+
+.download-button {
+  width: 210px;
+  height: 48px;
+  display: flex;
+  align-items: center;
+  background: var(--color-yellow);
+  margin: 0 auto;
+  border-radius: 50px;
+  justify-content: center;
+}
+
+
+/* ---------------------------------------------------------------------- *\
+                              #SECTION TESTIMONIALS
+\* ---------------------------------------------------------------------- */
+
+
+.testimonials {
+  max-width: 1440px;
+  height: 876px;
```

Comment:
No need to set height, when you implement content in section well, height will be right